app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -62,9 +62,11 @@ def deny_collection_access(exception)
       end
 
       def new
+        type_id = params[:type_id]
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.collections.new.header'), hyrax.new_dashboard_collection_path
+        add_breadcrumb t(:'hyrax.dashboard.collections.new.header'), request.path
+        @collection.collection_type_gid = CollectionType.find(type_id).gid unless type_id.nil?
```

Comment:
Prefer collection_type_id instead for type_id.  The type of the collection is considered to be its model which is type=Collection.  This might be confusing.